sycl/unittests/scheduler/FailedCommands.cpp

```diff
@@ -0,0 +1,61 @@
+//==----------- FailedCommands.cpp ---- Scheduler unit tests ---------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "SchedulerTest.hpp"
+
+#include <CL/cl.h>
+#include <CL/sycl.hpp>
+#include <detail/scheduler/scheduler.hpp>
+
+#include <gtest/gtest.h>
+
+using namespace cl::sycl;
+
+class MockCommand : public detail::Command {
+public:
+  MockCommand(detail::QueueImplPtr Queue)
+      : Command(detail::Command::ALLOCA, Queue) {}
+  void printDot(std::ostream &Stream) const override {}
+  void emitInstrumentationData() override {}
+  cl_int enqueueImp() override { return CL_SUCCESS; }
+};
+
+class TestScheduler : public detail::Scheduler {
```

Comment:
To align with the previous class?
```suggestion
class MockScheduler : public detail::Scheduler {
```